common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.cpp

```diff
@@ -458,6 +458,37 @@ bool base_wlan_hal_dwpal::ping()
     return true;
 }
 
+bool base_wlan_hal_dwpal::dwpal_send_cmd(const std::string &cmd, parsed_line_t &reply, int vap_id)
+{
+    if (!dwpal_send_cmd(cmd, vap_id)) {
+        return false;
+    }
+
+    if (strncmp(m_wpa_ctrl_buffer, "", 1)) {
```

Comment:
```suggestion
    if (m_wpa_ctrl_buffer[0] != '\0') {
```
(at least, for me that's easier to read)

Reply:
sure.